### ISSUE TITLE
Problem with condition of Metricbeat Value

### ISSUE THREAD

**Issue by alejandrobarrios07** [sirensolutions/sentinl#161]
We have an issue with load average metric of MetricBeat. We run the query and create a condition but doesn't receive the notification, instead write at the log:

"Condition Error for new_watcher_35zcqmjhk: SyntaxError: Unexpected number"

We have ELK version5.4.2

Thanks,
Alejandro

**Comment by lmangani** [comment]
Hi @alejandrobarrios07 
In order to help you we need to see your watcher, there's no way anyone can guess what you did.

**Comment by alejandrobarrios07** [comment]
Hi @lmangani , I'm new reporting issues, sorry.

Luckily I found the solution. When a metric name contains a number, the condition gives the error that remarks above.
My condition was:
`Payload.hits.hits [1] ._ source.system.load.1> 2`
And the solution I found was to process the metric by logstash and rename it by changing the number by the text, being as follows.
`Payload.hits.hits [1] ._ source.system.load.one> 2`

Thank you very much,

Alejandro
